wsd/AdminModel.cpp

```diff
@@ -1355,4 +1355,13 @@ void AdminModel::routeTokenSanityCheck()
     notify(oss.str());
 }
 
+void AdminModel::sendShutdownReceivedMsg()
+{
+    ASSERT_CORRECT_THREAD_OWNER(_owner);
+
+    std::ostringstream oss;
+    oss << "shutdown_received";
```

Comment:
Why not just `notify("shutdown_received");` ? Surely we don't need to stream a string literal then convert it to a string.